cmd/kube-apiserver/app/server.go

```diff
@@ -211,6 +213,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.AdmissionControlConfigFile, "admission-control-config-file", s.AdmissionControlConfigFile, "File with admission control configuration.")
 	fs.StringSliceVar(&s.EtcdServerList, "etcd-servers", s.EtcdServerList, "List of etcd servers to watch (http://ip:port), comma separated. Mutually exclusive with -etcd-config")
 	fs.StringVar(&s.EtcdConfigFile, "etcd-config", s.EtcdConfigFile, "The config file for the etcd client. Mutually exclusive with -etcd-servers.")
+	fs.StringSliceVar(&s.EtcdEventServerList, "etcd-event-servers", s.EtcdEventServerList, "List of etcd servers for events to watch (http://ip:port), comma separated. Mutually exclusive with -etcd-event-config")
```

Comment:
Could we just default to the event storage = normal storage for the time being?  I don't see there actually being an issue with having 2 interfaces backed into the same server lists.